api/graphql/schema.graphql

```diff
@@ -1654,16 +1657,22 @@ type Post {
   myEventResponse: String
   # The reactions of the logged-in user to this
   myReactions: [Reaction]
-  # Whether the logged-in user has upvoted the post or not
-  myVote: Boolean
   # Number of people that have one or more reactions to a post
   peopleReactedTotal: Int
   # Number of total reactions on the post
   postReactionsTotal: Int
   # Number of total "members" of the post. Used for Projects only right now
   postMembershipsTotal: Int
+  # A proposal's outcome: 'cancelled', 'quorum-not-met', 'in-progress', 'successful', 'tie'
```

Comment:
Looks like we never actually nailed down the possible outcomes. in the product spec it says it can set manually by the proposal creator, but it seems to me that a fixed set might be more useful? but we would definitely want a 'fail' option. Also what would a tie option mean? Seems like the outcome would either by a pass or a fail regardless of whether the number of votes in the options were tied

Reply:
Currently, outcomes are not truly used anywhere; they are not displayed and the value being different doesn't change any behavior of the platform.

What did we settle on? The notes for the product meeting aren't clear. At the moment, once the voting ends, its basically up to the user to figure out the winner (which is fairly clear). We could refine results once we have a version that is out for general use, allowing the public to have a stronger sense of what would work for them